dev/devicelab/bin/tasks/flutter_gallery_instrumentation_test.dart

```diff
@@ -16,9 +16,15 @@ Future<Null> main() async {
     final Directory galleryDirectory =
       dir('${flutterDirectory.path}/examples/flutter_gallery');
     await inDirectory(galleryDirectory, () async {
+      final Device device = await devices.workingDevice;
+      await device.unlock();
```

Comment:
Is it possible that the following steps take too long and the device locks again? Should unlock maybe happen right before we actually use the device?